src/less/blocks/layout.less

```diff
@@ -228,3 +228,17 @@
 .layout__row-double {
   margin: @space * 2 0;
 }
+
+.layout__row-fixed {
+  position: fixed;
+  z-index: 9;
+}
+
+.layout__row-group {
+  position: relative;
+  z-index: 10;
+}
+
+.layout__row-full {
+  min-height: 100vh;
+}
```

Comment:
new-line missing. please setup your editor to ALWAYS insert newline at the end of file